### ISSUE TITLE
No Parental Controls available

### ISSUE THREAD

**Issue by eugenialoli** [linuxmint/linuxmint#720]
### Distribution

Mint 22

### Package version

6.2.9

### Graphics hardware in use

Intel

### Frequency

Always

### Bug description

There is no parental control app. Now that Linux Mint is getting so popular, this is a must have. I didn't want to put this under a feature request because to me, in this day and age, especially for a home distro like Mint, it's a bug. 

A new admin app should be created that allows to restrict non-admin users:
- What hours they can login, and for how long
- Which apps they can access
- Which domains they're allowed to surf with a browser, with blanket allowance for chosen apps (e.g. zoom that might be used at school).

[edit, addition:] Make visible the Games category of apps only on specific days: Weekend, 5 pm Friday to 5pm Sunday (then there's study time), Saturday only, Sunday only.

### Steps to reproduce

There is no parental control app.

### Expected behavior

To exist.

**Comment by Jeremy7701** [comment]
If Mint is a home distro, just place the PC where it can be seen by the parent of the unfortunate user.
Alternatively, install software of your choice to hobble the PC.

**Comment by Secret-chest** [comment]
This could be useful. But it should respect a principle of Lm, that being no tracking. Also I don't see the use of time restriction, especially on a large device.

**Comment by eugenialoli** [comment]
Time restriction is one of the things parents ask. In many countries, kids have a study time, and a play time. It's during that play time that they're allowed to do things like using a computer. It should not be up to the parent to "give" the computer to the kid. Parents don't have the time (or might not be at home at the time) to specifically "give" their locked-away computer to their kids. Plus, the kids might need to use the computer for school and the parents might not be around. That's why time restrictions and availability are important. The kid would know when something is available and use it without fussing the parents.

So all three main features I mentioned in the first post are required. And I even forgot one more: Allow the usage of the Games category of apps only on specific days: Weekend, 5 pm Friday to 5pm Sunday (then there's study time), Saturday only, Sunday only. On the rest of the days, the parent can hide all the game apps from showing up in the menu.

But I agree with you, no tracking whatsoever. Just some simple rules on when and what to use.

If Mint implements this, it'd be the only Linux distro that does it properly. There are some third party utilities that can do similar things, but they're different apps for each of the three main features I described, and they're currently unmaintained AFAIK. No normal user coming to Linux from Windows is able to install and use these properly. Even Ubuntu doesn't have anything like it. Mint then becomes the go-to home computer distro. And with the onslaught of Linux adoption currently, that's a big plus for Mint's differentiation, IMHO.

**Comment by Jeremy7701** [comment]
Does your child have a mobile?
Is it locked down to prevent the more anti-social uses of social media?
I think not!
You shouldn't try to get a machine to do your parenting for you.

**Comment by eugenialoli** [comment]
No, they don't have a mobile phone at all, they don't need one. But they do need a computer for school project research.

**Comment by dewiweb** [comment]
@eugenialoli , for time restriction you can use [Timekpr-nExT](https://mjasnik.gitlab.io/timekpr-next/).

**Comment by eugenialoli** [comment]
Any news on forking the Gnome parental controls app at least? 

Timekpr-nExT is not doing what I asked. It only does 1/4th of what's needed.

**Comment by eugenialoli** [comment]
Also make sure the app fits on a 1366x768 resolution (with the cinnamon bar in place), because kids are mostly going to be given older laptops, and these usually ran that resolution (which is still very prevalent).
